docs/migrate-module.md

```diff
@@ -0,0 +1,100 @@
+1.  Confirm that you are creating a public repository in the terraform-ibm-modules organization and click **Create repository**.
+
+## Clone and initialize the repo
+
+Clone the repo that you just created.
+
+[inc-clone.md](inc-clone.md ':include')
+
+## Migrate the module code
+
+1.  From the root of the newly cloned repo, create a branch to work in by running the command `git checkout -b ＜YOUR_WORKING_BRANCH>`. Replace `＜YOUR_WORKING_BRANCH>` with your own branch name.
+s
+1.  Push the module code from the older repo to the new one by running these commands:
+    1.  Add the new repo as a remote to the older repo:
+
+        ```bash
+        git remote add origin2 https://github.com/terraform-ibm-modules/<REPO_NAME>
+        ```
+
+        Where `<REPO_NAME>` is your new repo. The command uses `origin2` because Git uses `origin` by default.
+    1.  Check that the remote points from your old repo to the new one by running this command:
+
+        ```bash
+        git remote -v
+        ```
+    1.  Push the module code from the older repo to the main branch of the new repo:
+        ```bash
+        git push origin2 main
+        ```
+
+## Add the newer CI code
+
+Copy the following files from the `.github` directory of the terraform-ibm-module-template.
+
+?> **Tip:** You can use the [migration script](https://github.com/terraform-ibm-modules/common-dev-assets/blob/main/repo_migration.sh) in the `common-dev-assets` repo to add the CI code.
+
+1.  Browse to the [terraform-ibm-module-template](https://github.com/terraform-ibm-modules/terraform-ibm-module-template/blob/main/.github) `.github` directory.
+1.  Copy the `settings.yml` file.
+1.  Copy the `workflows` directory that contains the GitHub Action workflow
+
+## Make updates to pass the pre-commit checks
+
+1.  Run the pre-commit hooks from the root of the newly cloned repo to identify other changes that you need to make:
+
+    ```bash
+    pre-commit run --all-files
+    ```
+
+1.  Iteratively address all the issues from this check, commit the changes, and run the `pre-commit` command until you have a clean check.
+
+## Create or update examples
+
+Create or update at least one end-to-end example to test your code changes. You can see the structure in the [examples](https://github.com/terraform-ibm-modules/terraform-ibm-module-template/tree/main/examples) directory of the terraform-ibm-module-template.
+
+?> **Tip:** You can use the [migration script](https://github.com/terraform-ibm-modules/common-dev-assets/blob/main/repo_migration.sh) in the `common-dev-assets` repo to add the CI code.
+
+1.  Implement (or update) the logic for your module examples by updating the `main.tf`, `version.tf`, `variables.tf`, and `version.tf` Terraform files in the `examples/default` directory.
```

Comment:
version.tf is listed twice

Reply:
provider.tf is probably what was supposed to be listed instead of version.tf twice

Reply:
it was outputs.

Reply:
provider.tf is generally needed in examples too, as it's technically a root level module. So maybe add that too

Reply:
added `provider.tf`